H/H.cabal

```diff
@@ -1,35 +1,38 @@
-name:                H
-version:             0.9.0.1
-license:             BSD3
-license-file:        LICENSE
-copyright:           Copyright (c) 2013-2015 Amgen, Inc.
-                     Copyright (c) 2015-2018 Tweag I/O Limited.
-author:              Mathieu Boespflug, Facundo Dominguez, Alexander Vershilov
-maintainer:          Mathieu Boespflug <m@tweag.io>
-build-type:          Simple
-Category:            FFI
-Synopsis:            The Haskell/R mixed programming environment.
-description:         This package is part of the HaskellR project.
-homepage:            https://tweag.github.io/HaskellR
-cabal-version:       >=1.10
-extra-source-files:  H.ghci
+cabal-version:      >=1.10
+name:               H
+version:            0.9.0.1
+license:            BSD3
+license-file:       LICENSE
+copyright:
+    Copyright (c) 2013-2015 Amgen, Inc.
+    Copyright (c) 2015-2018 Tweag I/O Limited.
+
+maintainer:         Mathieu Boespflug <m@tweag.io>
+author:             Mathieu Boespflug, Facundo Dominguez, Alexander Vershilov
+homepage:           https://tweag.github.io/HaskellR
+synopsis:           The Haskell/R mixed programming environment.
+description:        This package is part of the HaskellR project.
+category:           FFI
+build-type:         Simple
+extra-source-files: H.ghci
 
 source-repository head
-  type:     git
-  location: git://github.com/tweag/HaskellR.git
-  subdir: H
+    type:     git
+    location: git://github.com/tweag/HaskellR.git
+    subdir:   H
 
 executable H
-  main-is:             H.hs
-  other-modules:       Paths_H
-  build-depends:       base >= 4.6 && < 5
-                     , bytestring >= 0.10
-                     , cmdargs >= 0.10.5
-                     , file-embed >= 0.0.7
-                     , inline-r >= 0.9 && < 0.11
-                     , pretty >= 1.1
-                     , process >= 1.2
-                     , temporary >= 1.2.0.3
-                     , vector >= 0.10
-  default-language:    Haskell2010
-  ghc-options:         -Wall -threaded
+    main-is:          H.hs
+    other-modules:    Paths_H
+    default-language: Haskell2010
+    ghc-options:      -Wall -threaded
+    build-depends:
+        base >=4.6 && <5,
+        bytestring >=0.10 && <0.12,
+        cmdargs >=0.10.5 && <0.11,
```

Comment:
worth using the caret operator where it fits? e.g. `cmdargs ^>= 0.10.5`?

Reply:
I need to bump `cabal-version` for that. Which would be fine I guess.